scripts/install/opt/seagate/cortx/motr/libexec/motr_data_recovery.sh

```diff
@@ -871,6 +919,28 @@ EOF
             # restart the execution of command if exit code is ESEGV error
             while [[ $cmd_exit_status == $ESEGV ]];
             do
+                #Following is the code to limit the number of core-m0beck and m0trace files, generated due to m0beck crash( receving SEGV signal ), to 2 each.
+                core_m0beck_file_count=$(cd /var/log/crash; find . -type f -exec stat  -c "%n %y" {} \;| sort -n | grep core-m0beck | awk '{if($2>"'"${SCRIPT_START_DATE}"'") print $2" "$3; else if($2=="'"${SCRIPT_START_DATE}"'" && $3>="'"${SCRIPT_START_TIME}"'") print $2" "$3; else echo }' | wc -l)
```

Comment:
How did this differentiate core dump file   for ocal  and remote m0-beck ?
 If not that there is a chance we lost  core dump one of the local node or remote node beck

Reply:
@RakeshVaghasiya No, this code wont differentiate between local node m0beck and remote node m0beck.

This can be achieved by 2 ways
1. Creating another folder inside crash where we exclusively put those core files which have been generated due to running of recovery on remote node from local node.
2. We can modify the SCRIPT_START_TIME and SCRIPT_START_DATE variable to get the files that have been generated after we enter the else part of the code, but this wont be good option as local node becktool and remote node becktool start their processing in parallel.